secrets/index.js

```diff
@@ -0,0 +1,10 @@
+const secretsMap = {}
+
+module.exports = {
+  getSecret: function (key) {
+    return secretsMap[key] || process.env['WB_SECRET_' + key]
```

Comment:
As a quick note, we should try to avoid any references to Windbreaker in the code. Eventually, I kinda want this to be a generic repo that contains utilities for building services.